go.sum

```diff
@@ -1238,6 +1238,7 @@ golang.org/x/sys v0.0.0-20190916202348-b4ddaad3f8a3/go.mod h1:h1NjWce9XRLGQEsW7w
 golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20191001151750-bb3f8db39f24/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
 golang.org/x/sys v0.0.0-20191005200804-aed5e4c7ecf9/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
```

Comment:
Do you have to add this manually? wondering why it is not generated automatically by go mod.

Reply:
yes @yuzisun

Reply:
@Suresh-Nakkeran looks like there is some behavior change in go 1.17, `go run k8s.io/kube-openapi/cmd/openapi-gen` in `hack/update-openapigen.sh` no longer pull the go mod dependency and you need to run `go get k8s.io/kube-openapi/cmd/openapi-gen` first to update `go.mod`.

Reply:
Also I think we need to generate the python SDK from the updated `swagger.json`